specification/compute/resource-manager/readme.md

```diff
@@ -34,7 +34,7 @@ These are the global settings for the Compute API.
 title: ComputeManagementClient
 description: Compute Client
 openapi-type: arm
-tag: package-2020-12-01
+tag: package-2021-03-01
```

Comment:
If you want to include this in the 2021-03-01 release, then you should make this PR against the branch https://github.com/Azure/azure-rest-api-specs/tree/compute-2021-03

Reply:
resolved as per email discussion